framework/SwiftPlot/PlotMarkers.swift

```diff
@@ -1,8 +1,8 @@
 public struct PlotMarkers{
-    public var xMarkers = [Point]()
-    public var yMarkers = [Point]()
-    public var xMarkersTextLocation = [Point]()
-    public var yMarkersTextLocation = [Point]()
+    public var xMarkers = [Pair<FloatConvertible,FloatConvertible>]()
```

Comment:
Another place where a specialized Point type might make this a little simpler. Also, you're again treating FloatConvertible as a type, where you'd probably mean Float directly.